libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/MainActionStrip.kt

```diff
@@ -94,14 +92,6 @@ class MainActionStrip(
         return PlacesListOnMapScreen(
             SearchCarContext(mainCarContext),
             placesProvider,
-            PlacesListItemMapper(
-                PlaceMarkerRenderer(mainCarContext.carContext),
-                mainCarContext
-                    .mapboxNavigation
```

Comment:
It's this dependency that needs to be moved, getting the unit type to become observable is a bit complicated